app/models/trainee.rb

```diff
@@ -185,4 +185,13 @@ def available_courses
   def clear_disabilities
     disabilities.clear
   end
+
+  def get_award_type
```

Comment:
I wonder whether this should be handled by the `TrainingRouteManager` - that's where we're currently keeping logic that's based on the trainee's training route.